src/Albany_ModelEvaluator.cpp

```diff
@@ -619,28 +617,49 @@ Thyra_OutArgs ModelEvaluator::createOutArgsImpl() const
     bool hess_vec_prod_g_px_support = aDHessVec;
     bool hess_vec_prod_g_pp_support = aDHessVec;
 
+    bool hess_vec_prod_f_xx_support = aDHessVec;
```

Comment:
Should not this be false, since we are not yet providing the AD hessian for f?

Reply:
You are right, this is not correct yet.
This is an error which comes from the fact that the commits of this PR branch came from a branch were I am developing the AD Hessian-vector products for f.

Sorry for this mistake.